test/Conversion/TorchToLinalg/elementwise.mlir

```diff
@@ -84,3 +84,28 @@ func @elementwise$static_1(%arg0: !torch.vtensor<[?],f32>, %arg1: !torch.vtensor
   %1 = torch.aten.mul.Tensor %arg0, %arg1 : !torch.vtensor<[?],f32>, !torch.vtensor<[1],f32> -> !torch.vtensor<[?],f32>
   return %1 : !torch.vtensor<[?],f32>
 }
+
+// CHECK-LABEL: func @elementwise$to_dtype
```

Comment:
I think we don't need this unit test. There is no new code logic related to general elementwise ops and the operation specific logic are covered by the e2e test already.